website/docs/developer/research/research-lambda-container.md

```diff
@@ -0,0 +1,139 @@
+---
+id: research-lambda-container
+title: Using Lambda functions as container research Notes
+description: Research Notes on 
+---
+
+## Overview
+
+Lambda functions can now be deployed as container images using Docker instead of zip files. This research webpage discusses how lambda functions can be prototyped as container and its pros and cons.
+
+## Pros and Cons of using container for lambdas
```

Comment:
The context required to understand this is later in the document. Recommend moving above Future Direction.

Reply:
I think keeping the `pros and cons` section before the prototype section will  give a high level overview before going into details.

Reply:
I feel that so long as this includes references to technical aspects such as ECR, it requires additional explanation.

Reply:
I see that, but since it is later in the document, I still feel like this is out of order.

Reply:
okay I think creating an ECR section would be better in this case. 

added